backend/app/controllers/api/v1/environment_open_api.rb

```diff
@@ -37,6 +37,10 @@ class Api::V1::EnvironmentOpenApi
         key :type, :boolean
         key :readOnly, true
       end
+      property :is_country_eligible do
+        key :type, :boolean
+        key :readOnly, true
+      end
       property :accounts_env_name do
```

Comment:
I'd suggest renaming this to be "is_eligible" so the check will remain valid if we later take other things into account, such as states perhaps